scirpy/io/_io.py

```diff
@@ -663,7 +663,9 @@ def to_dandelion(adata: AnnData):
 
 
 @_doc_params(doc_working_model=doc_working_model)
-def from_dandelion(dandelion, transfer=False) -> AnnData:
+def from_dandelion(
+    dandelion, transfer: bool = False, include_fields: Optional[Collection[str]] = None
```

Comment:
Now that the bug is fixed, I don't think we require any special casing.
To ensure `from_dandelion` and `read_airr` work in a consistent way, how about just passing `**kwargs` to `read_airr`. 

In that case, per default, `DEFAULT_AIRR_FIELDS` get imported from dandelion, if the user wants all, they can just pass include_fields=None.

Reply:
sounds great. I'll sort that out.

Reply:
Hmm if i remove the special casing, it stops working. damn it.